frontend-new/next.config.js

```diff
@@ -0,0 +1,19 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  // reactStrictMode: true,
+  // swcMinify: true,
+  async rewrites() {
```

Comment:
These should come from environment variables.

Reply:
it is just for development, I think we can change it when we integrate things e2e